GameServer/ai/brain/Animist/TurretBrain.cs

```diff
@@ -49,16 +49,18 @@ public override int ThinkInterval
 		/// </summary>
 		public override void Think()
 		{
-		  GamePlayer playerowner = GetPlayerOwner();
-			if (playerowner != null && (GameTimer.GetTickCount() - playerowner.Client.GameObjectUpdateArray[new Tuple<ushort, ushort>(Body.CurrentRegionID, (ushort)Body.ObjectID)]) > ThinkInterval)
-		  {
-			playerowner.Out.SendObjectUpdate(Body);
-		  }
-
-		  if(!CheckSpells(eCheckSpellType.Defensive))
-		  {
-		  	AttackMostWanted();
-		  }
+            GamePlayer playerowner = GetPlayerOwner();
+            var gameObjectRegionAndId = new Tuple<ushort, ushort>(Body.CurrentRegionID, (ushort)Body.ObjectID);
+            var isObjectInUpdateArray = playerowner.Client.GameObjectUpdateArray.TryGetValue(gameObjectRegionAndId, out var updateTime);
+            if (playerowner != null && isObjectInUpdateArray && (GameTimer.GetTickCount() - updateTime > ThinkInterval))
+            {
+                playerowner.Out.SendObjectUpdate(Body);

```

Comment:
If I'm not mistaken, I didn't change the behavior there and I didn't really question the motivation of that since I actually only wanted to fix the run-through bug. The 1.125 fix (the highlighted code) was actually even only a by-product of that.